app/components/concerns/op_turbo/streamable.rb

```diff
@@ -66,7 +66,7 @@ def render_as_turbo_stream(view_context:, action: :update)
       def insert_as_turbo_stream(component:, view_context:, action: :append)
         template = component.render_in(view_context)
 
-        unless @component_wrapper_used
+        unless component.instance_variable_get(:@component_wrapper_used)
```

Comment:
Instead of accessing the private instance variable, the variable needs to be exposed e.g. via  a `wrapped?` method.